docs/releases/v8_20.md

```diff
@@ -0,0 +1,39 @@
+- [**DietPi-Software**](../../dietpi_tools/software_installation/#dietpi-software) | [**DietPi-LetsEncrypt**](../../software/system_security/#lets-encrypt) :octicons-arrow-right-16: Resolved a DietPi v8.19 regression where applying the HTTPS certificate for Lighttpd fails. Many thanks to @midniteca for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6460>
+- [**DietPi-Tools**](../../dietpi_tools/) | [**DietPi-Config**](../../dietpi_tools/system_configuration/#dietpi-config) :octicons-arrow-right-16: Resolved an issue where enabling Bluetooth on SBCs with Armbian firmware failed, because of a conflict between `armbian-firmware` and `bluez-firmware` packages. `bluez-firmware` will not be tried to be installed anymore if `armbian-firmware` is. Many thanks to [@innovodev](https://dietpi.com/forum/u/innovodev){: class="nospellcheck"} for reporting this issue: <https://dietpi.com/forum/t/upgrading-dietpi-from-bullseye-to-bookworm/15963/16>
+- [**DietPi-Software**](../../dietpi_tools/software_installation/#dietpi-software) | [**Lighttpd**](../../software/webserver_stack/#lighttpd) :octicons-arrow-right-16: Resolved a DietPi v8.19 regression where the installation of Pi-hole, ownCloud, Nextcloud, Pydio and MediaWiki failed if Lighttpd was selected as webserver and HTTPS not yet enabled via `dietpi-letsencrypt`. Many thanks to @bruno-briner for reporting this issue: <https://github.com/MichaIng/DietPi/issues/6455>
+- [**DietPi-Software**](../../dietpi_tools/software_installation/#dietpi-software) | [**PaperMC**](../../software/gaming/#papermc) :octicons-arrow-right-16: Resolved an issue where the install failed when selecting the `Geyser` and `Floodgate` plugins, as the download URL changed. Many thanks to @NatureHog for reporting and fixing this issue: <https://github.com/MichaIng/DietPi/issues/6471>
+- [**DietPi-Software**](../../dietpi_tools/software_installation/#dietpi-software) | [**vaultwarden**](../../software/cloud/#vaultwarden) :octicons-arrow-right-16: Worked around an issue where since Bookworm the service could have failed to start if DietPi userdata were moved to an external drive. The update, which includes a bump to latest vaultwarden v1.29.1 and web vault v2023.5.0 will be applied automatically as part of the DietPi update. Many thanks to [@huidbui25](https://dietpi.com/forum/u/huidbui25) for reporting this issue: <https://dietpi.com/forum/t/automount-option-in-fstab-prevents-automatically-mounting-a-partition-in-due-time-on-bookworm/17463>
```

Comment:
Forgot the spellcheck exclusion.
```suggestion
- [**DietPi-Software**](../../dietpi_tools/software_installation/#dietpi-software) | [**vaultwarden**](../../software/cloud/#vaultwarden) :octicons-arrow-right-16: Worked around an issue where since Bookworm the service could have failed to start if DietPi userdata were moved to an external drive. The update, which includes a bump to latest vaultwarden v1.29.1 and web vault v2023.5.0 will be applied automatically as part of the DietPi update. Many thanks to [@huidbui25](https://dietpi.com/forum/u/huidbui25){: class="nospellcheck"} for reporting this issue: <https://dietpi.com/forum/t/automount-option-in-fstab-prevents-automatically-mounting-a-partition-in-due-time-on-bookworm/17463>
```
We really need to find a way to shortcut these forum profile links 😄.

Reply:
I have to remind that when doing the releasenotes. Possibly a kind of "releasenotes checklist" with the operations to do would be beneficial. :-)

Reply:
Probably we can prefix the forum usernames differently within the `CHANGELOG.txt`, so no `@` but with `!` or something like that, to have a better marker. This also solves the issue that I always need to wrap them into code tags on GitHub to not ping possibly existing GitHub accounts.

Reply:
Would be an idea. The spellcheck will guide one to check the user link.

Reply:
Right, at least in most cases. And with a unique marker we should also able to find/write a plugin which replaces them with forum links automatically. Probably `@!Name` or `@:Name` is better, as it more unique/less likely to be meant differently.